src/shared/variables.css

```diff
@@ -1,9 +1,11 @@
+@import './colors.css';
+
 :root {
   /* Colors */
-  --box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.2);
+  --rvr-box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.2);
 
   /* Fonts */
-  --base-font-color: #58666e;
-  --base-font-family: 'Source Sans Pro', sans-serif;
-  --fontFamily-mono: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono", "Courier New", monospace;
+  --rvr-base-font-color: #58666e;
+  --rvr-base-font-family: 'Source Sans Pro', sans-serif;
```

Comment:
Should we go ahead and switch this to the Proxima font while we are updating this?

Reply:
Yes, but no. All the colors have to get updated too, I just don't want to mix concerns with this PR any more than they already are.